recipes/kronprod/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "kronprod" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Jbwasse2/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 15323aad0af5a392da9dc5d30943ab5e2bd4a1d83a45ec282209e2dee12e3b5a
+
+build:
+  noarch: python
```

Comment:
This recipe cannot be ``noarch``, please remove it
As you depend on a platform (in this case some compilers), this recipe can't be independent of a platform

Reply:
Done!